systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -4,6 +4,16 @@
 
 ![Alt Text](./diagrams/05-building-blocks.drawio.svg)
 
+
+|  Building Block | Type/Context  | Responsibility   |
```

Comment:
Let's just get rid of the Type/Context column if it doesn't make sense for this diagram!